recipes-scanners/clamav/clamav_1.4.1.bb

```diff
@@ -0,0 +1,226 @@
+SUMMARY = "ClamAV anti-virus utility for Unix - command-line interface"
+DESCRIPTION = "ClamAV is an open source antivirus engine for detecting trojans, viruses, malware & other malicious threats."
+HOMEPAGE = "http://www.clamav.net/index.html"
+SECTION = "security"
+LICENSE = "LGPL-2.1-only"
+
+DEPENDS = "glibc llvm libtool db openssl zlib curl libxml2 bison pcre2 json-c libcheck rust-native cargo-native libmspack"
+ 
+COMPATIBLE_HOST:libc-musl:class-target = "null"
```

Comment:
Try to follow the variable ordering styleguide.

https://docs.yoctoproject.org/dev/contributor-guide/recipe-style-guide.html#recipe-ordering